test/compilable/test1537.d

```diff
@@ -25,44 +25,6 @@ static assert( __traits(identifier, bug4732) == "bug4732");
 
 /**************************************/
 
-template Compileable(int z) { bool OK=true;}
-
-int bug5245a(U)()
-{
-    { enum   T { a = 5  } T v; }
-    { enum   T { a = 6  } T w; }
-    return 91;
-}
-
-int bug5245b(U)()
-{
-    { struct T { int a = 2; } T v; }
-    { union  T { int a = 3; } T w; }
-    return 91;
-}
-
-int bug5245c(U)()
-{
-    { struct T { int a = 2; } T v; }
-    { class  T { int a = 3; } T w; }
-    return 91;
-}
-
-int bug5245d(U)()
-{
-    { enum  T { a = 3 } T w; }
-    { struct T { int a = 2; } T v; }
-    return 91;
-}
-
-
-static assert(!is(typeof(Compileable!(bug5245a!(int)()).OK)));
```

Comment:
It would be good to keep these tests around with a comment saying as of [bugzilla reference], test is now compilable.

Reply:
yeah but I dont know. I don't even know why they were here. They looked like a primitive way to do something like `__traits(compiles)`.

Reply:
Yes, this predates `__traits`, but don't adapt it, as it is still a code path that should remain tested.

Reply:
to be clear you want me to change `!is` into `is`, right ?

Reply:
okay done

Reply:
Yes.